src/NDS.h

```diff
@@ -469,6 +469,8 @@ class NDS
     virtual void ARM7IOWrite16(u32 addr, u16 val);
     virtual void ARM7IOWrite32(u32 addr, u32 val);
 
+    void SetMemoryExtension(bool enabled);
```

Comment:
Could this flag be extended to also work on TWL/DSi? Its debug units have 32MB of RAM as opposed to 16MB of RAM; it's available at 0xC000000 ~ 0xDFFFFFF.